src/PhpWord/Writer/HTML/Element/Title.php

```diff
@@ -49,7 +50,68 @@ public function write()
             $text = $writer->write();
         }
 
-        $content = "<{$tag}>{$text}</{$tag}>" . PHP_EOL;
+        $content = "<{$tag}>";
+        $content .= $this->writeTrackChangeOpening();
+        $content .= "{$text}";
+        $content .= $this->writeTrackChangeClosing();
+        $content .= "</{$tag}>" . PHP_EOL;
+
+        return $content;
+    }
+
+    /**
+     * writes the track change opening tag
+     *
+     * @return string the HTML, an empty string if no track change information
+     */
+    private function writeTrackChangeOpening()
```

Comment:
should avoid code duplication and move those 2 methods to the parent AbstractElement